cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -147,7 +147,127 @@ property replaces the ``schedule`` property of an ETL Batch Application.
 
 In the example code above, we will use a *ProjectionTransform* (a type of Transform) to drop certain
 columns in the incoming data. A *StreamSink* in the final step needs a data field property
-that it will use as the content for the data to be written. 
+that it will use as the content for the data to be written.
+
+DAG in transform
+----------------
+
+ETLConfig support DAG in pipeline, which allows non-linear execution of pipeline stages.
+
+Example :
+pipeline reads from the stream ``purchaseStats``. It replicates the stream events to the table ``replicaTable``,
+while it only writes the userIds to ``usersTable`` when a user's purchase price is greater that $1000,
+this filtering logic is applied using the script ``spendingUsersScript``.
+
```

Comment:
Should we add something pointing to the "connections" section, as that is what is creating the non-linear execution?